addons/packages/harbor/2.2.3/test/go.sum

```diff
@@ -1,3 +1,4 @@
+bazil.org/fuse v0.0.0-20160811212531-371fbbdaa898 h1:SC+c6A1qTFstO9qmB86mPV2IpYme/2ZoEQ0hrP+wo+Q=
```

Comment:
I just checked out to this PR using `gh pr checkout 2444` and ran `go mod tidy` inside `addons/packages/harbor/2.2.3/test` and saw that there was a diff in `git status` and `git diff`

Reply:
I think we can solve the problem of out of sync / outdated `go.mod` and `go.sum` files like this - https://github.com/vmware-tanzu/community-edition/pull/2167#issuecomment-935675031

Reply:
Created an issue to discuss this #2453